site/en/docs/devtools/recorder/extensions/index.md

```diff
@@ -1,40 +1,62 @@
 ---
 layout: "layouts/doc-post.njk"
-title: "Recorder extensions"
+title: "Extensions"
 authors:
   - sofiayem
   - jecelynyeen
 date: 2023-02-22
 updated: 2023-02-22
-description: "A list of known Recorder extensions"
+description: "A list of known Recorder extensions."
 tags:
   - test
   - performance
 ---
 
+This page lists knows Chrome extensions for the DevTools **Recorder**.
+
 ## Export extensions
 
 {% Aside %}
 **Note**: This feature is available from Chrome version 104.
 {% endAside %}
 
-Export extensions allow customizing the code generated by the Recorder panel:
+Export extensions let you to customize the code generated by the **Recorder** panel:
```

Comment:
I don't think we need extra text for every extension. I feel it is redundant since all extensions do the same thing. Any other opinions?

Reply:
I just saw this comment. I dunno. We can kinda promote third-party projects by providing more links. I reorg'd the list into a glossary, it read better.

Reply:
WDYT?

Reply:
That sounds fine by me but I have two concerns as a user:

1) I can read detailed descriptions on the web store page if I am interested in a new tool. If I already know the tools or use specific tools, then the information is redundant. Additional text content requires more time to read through the list to see if your favourite tools are supported.
2) maintaining the text description would be more work. And if we the vendors don't like them anymore or want to update slogans slogans for whatever reason, it will require a manual update unlike the webstore that offers the self-service UI.

Reply:
I like how the glossary looks visually though! so LGTM.